### ISSUE TITLE
[dev.icinga.com #3847] API does not update current_state to Pending when has_been_checked==0

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga-web#1046]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/3847

**Created by mfrosch on 2013-03-13 09:25:03 +00:00**

Assignee: _mfrosch_
Status: _Rejected_  (closed on _2013-03-13 10:43:51 +00:00_)
Target Version: _(none)_
Last Update: _2013-03-13 10:43:51 +00:00 (in Redmine)_

    Icinga Version: 1.8.4
    Icinga Web Version: 1.9.0-dev
    IDO Version: 1.8.4
    OS Version: Debian/wheezy
    DB Type: MySQL
    DB Version: all
    Browser Version: all

---

Due to a problem with the API the current\_state of a host or service is not updated to PENDING

Basically this only works when the field has\_been\_checked is requested via the API.
